cmd/runtimetest/main.go

```diff
@@ -630,6 +630,54 @@ func validateMountsExist(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateMountsOrder(spec *rspec.Spec) error {
+	if runtime.GOOS == "windows" {
+		return nil
```

Comment:
Possibly warn that the check is not implemented (like we currently do [here][1])?

[1]: https://github.com/opencontainers/runtime-tools/blob/9e0e42dbf918070406a2a4a2e1476e7350ba9129/validate/validate.go#L334